src/rpc/blockchain.cpp

```diff
@@ -784,8 +784,8 @@ static RPCHelpMan getblockfrompeer()
         "Subsequent calls for the same block and a new peer will cause the response from the previous peer to be ignored.\n"
         "\nReturns an empty JSON object if the request was successfully scheduled.",
         {
-            {"block_hash", RPCArg::Type::STR_HEX, RPCArg::Optional::NO, "The block hash to try to fetch"},
-            {"peer_id", RPCArg::Type::NUM, RPCArg::Optional::NO, "The peer to fetch it from (see getpeerinfo for peer IDs)"},
+            {"blockhash", RPCArg::Type::STR_HEX, RPCArg::Optional::NO, "The block hash to try to fetch"},
+            {"nodeid", RPCArg::Type::NUM, RPCArg::Optional::NO, "The peer to fetch it from (see getpeerinfo for node IDs)"},
```

Comment:
I might be missing something, but the other, existing, RPC is called get*peer*info and not get*node*info. Also, the result is called "peer index".

```
getpeerinfo ... Result:
[                                         (json array)
  {                                       (json object)
    "id" : n,                             (numeric) Peer index